filebeat/etc/beat.yml

```diff
@@ -140,6 +140,10 @@ filebeat:
       # file is skipped, as the reading starts at the end. We recommend to leave this option on false
       # but lower the ignore_older value to release files faster.
       #force_close_files: false
+      json_decoder:
```

Comment:
Not sure if we perhaps should just call it just `json` instead of json_decoder. It is shorter and will not get us into the discussion of adding further "decoders" :-)